AzureMonitorAgent/agent.py

```diff
@@ -1194,7 +1194,8 @@ def is_vm_supported_for_extension(operation):
                        'debian' : ['9', '10'], # Debian
                        'ubuntu' : ['16.04', '18.04', '20.04'], # Ubuntu
                        'suse' : ['12'], 'sles' : ['15'], # SLES
-                       'cbl-mariner' : ['1'] # Mariner
+                       'cbl-mariner' : ['1'], # Mariner 1.0
+                       'mariner' : ['2.0'] # Mariner 2.0
```

Comment:
is it always `2.0` (vs 2); do we want to leave as 2 like with mariner 1? We match the prefix, so 2 will still match 2.0

Reply:
I didn't realize we match the prefix - changing it now, thank you for catching this!